google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -193,7 +193,7 @@ clouddebug.errorset=The snapshot location could not be set.
 clouddebug.continuesession=Continue Session
 clouddebug.continueanyway=Continue Anyway
 clouddebug.text=Google Stackdriver Debug
-clouddebug.stackdriver.debugger=Stackdriver Debugger
+clouddebug.stackdriver.debugger=Google Stackdriver Debugger
```

Comment:
Good point. It's going to not be, but then be again by #1078 and anything that wants to use it in the meantime.